diff --git a/src/Microsoft.DotNet.Wpf/src/PresentationUI/PresentationUI.csproj b/src/Microsoft.DotNet.Wpf/src/PresentationUI/PresentationUI.csproj
index 93d30ed..21f652c 100644
--- a/src/Microsoft.DotNet.Wpf/src/PresentationUI/PresentationUI.csproj
+++ b/src/Microsoft.DotNet.Wpf/src/PresentationUI/PresentationUI.csproj
@@ -1,4 +1,4 @@
-﻿<Project>
+<Project>
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   <PropertyGroup>
     <Platforms>x86;x64;arm64</Platforms>
@@ -21,29 +21,29 @@
     <Page Include="TenFeetInstallationError.xaml" />
     <Page Include="TenFeetInstallationProgress.xaml" />
     <Page Include="MS\Internal\Documents\FindToolBar.xaml" />
-    <Page Include="themes\MediaCenterTheme.xaml" />
-    <Page Include="themes\Aero.NormalColor.xaml">
+    <Page Include="Themes\MediaCenterTheme.xaml" />
+    <Page Include="Themes\Aero.NormalColor.xaml">
       <GenerateTheme>true</GenerateTheme>
     </Page>
-    <Page Include="themes\Aero2.NormalColor.xaml">
+    <Page Include="Themes\Aero2.NormalColor.xaml">
       <GenerateTheme>true</GenerateTheme>
     </Page>
-    <Page Include="themes\AeroLite.NormalColor.xaml">
+    <Page Include="Themes\AeroLite.NormalColor.xaml">
       <GenerateTheme>true</GenerateTheme>
     </Page>
-    <Page Include="themes\Classic.xaml">
+    <Page Include="Themes\Classic.xaml">
       <GenerateTheme>true</GenerateTheme>
     </Page>
-    <Page Include="themes\Generic.xaml">
+    <Page Include="Themes\Generic.xaml">
       <GenerateTheme>true</GenerateTheme>
     </Page>
-    <Page Include="themes\Luna.Homestead.xaml">
+    <Page Include="Themes\Luna.HomeStead.xaml">
       <GenerateTheme>true</GenerateTheme>
     </Page>
-    <Page Include="themes\Luna.Metallic.xaml">
+    <Page Include="Themes\Luna.Metallic.xaml">
       <GenerateTheme>true</GenerateTheme>
     </Page>
-    <Page Include="themes\Luna.NormalColor.xaml">
+    <Page Include="Themes\Luna.NormalColor.xaml">
       <GenerateTheme>true</GenerateTheme>
     </Page>
   </ItemGroup>
@@ -169,9 +169,7 @@
     <ProjectReference Include="$(WpfSourceDir)System.Xaml\System.Xaml.csproj" />
     <ProjectReference Include="$(WpfSourceDir)PresentationFramework\PresentationFramework.csproj" />
     <ProjectReference Include="$(WpfSourceDir)WindowsBase\WindowsBase.csproj" />
-    <ProjectReference Include="$(WpfSourceDir)System.Printing\System.Printing.vcxproj">
-      <UndefineProperties>TargetFramework;TargetFrameworks</UndefineProperties>
-    </ProjectReference>
+    <ProjectReference Include="$(WpfSourceDir)System.Printing\ref\System.Printing-ref.csproj" />
     <ProjectReference Include="$(WpfSourceDir)ReachFramework\ReachFramework.csproj" />
     <ProjectReference Include="$(WpfSourceDir)UIAutomation\UIAutomationProvider\UIAutomationProvider.csproj" />
     <ProjectReference Include="$(WpfSourceDir)UIAutomation\UIAutomationTypes\UIAutomationTypes.csproj" />
diff --git a/src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero/PresentationFramework.Aero.csproj b/src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero/PresentationFramework.Aero.csproj
index 3ba1e3c..742fd7a 100644
--- a/src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero/PresentationFramework.Aero.csproj
+++ b/src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Aero/PresentationFramework.Aero.csproj
@@ -1,4 +1,4 @@
-﻿<!-- PresentationFramework.Aero -->
+<!-- PresentationFramework.Aero -->
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
@@ -20,7 +20,7 @@
 
   <!-- Compile Targets -->
   <ItemGroup>
-    <Page Include="Themes\Aero.NormalColor.xaml">
+    <Page Include="themes\Aero.NormalColor.xaml">
       <Generator>MSBuild:Compile</Generator>
     </Page>
     <Compile Include="$(WpfSharedDir)\RefAssemblyAttrs.cs" />
